code/modules/bomberman/bomberman.dm

```diff
@@ -878,6 +878,10 @@ var/global/list/arenas = list()
 
 		message_admins("[key_name_admin(user.client)] created a \"[size]\" Bomberman arena at [center.loc.name] ([center.x],[center.y],[center.z]) (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[center.x];Y=[center.y];Z=[center.z]'>JMP</A>)")
 		log_game("[key_name_admin(user.client)] created a \"[size]\" Bomberman arena at [center.loc.name] ([center.x],[center.y],[center.z]) ")
+
+		for(var/mob/dead/observer/O in observers)
```

Comment:
The observers list is defined, but I'm not sure its actually ever used so I hope you tested this